src/FINDOLOGIC/Export/Data/Attribute.php

```diff
@@ -68,16 +68,14 @@ public function getDomSubtree(\DOMDocument $document)
     /**
      * @inheritdoc
      */
-    public function getCsvFragment()
+    public function getCsvFragment(array $availableProperties = [])
     {
-        $encoded = '';
+        $attributeParts = [];
 
         foreach ($this->values as $value) {
-            $encoded .= sprintf('%s=%s&', urlencode($this->key), urlencode($value));
+            $attributeParts []= sprintf('%s=%s', urlencode($this->key), urlencode($value));
```

Comment:
`$attributeParts []=` should be `$attributeParts[] =`

Reply:
It hurts a little to do this, because of `+=`, `-=`, `.=` and others, but as I understand it what you're suggesting is the generally understood convention, so I'll go with it.

:heavy_check_mark: